pages/glossary/included-in-the-accessibility-tree.md

```diff
@@ -3,8 +3,21 @@ title: Included in the accessibility tree
 key: included-in-the-accessibility-tree
 ---
 
-Elements included in the accessibility tree of platform specific accessibility APIs. Elements in the accessibility tree are exposed to assistive technologies, allowing users to interact with the elements in a way that meet the requirements of the individual user.
+Elements included in the accessibility tree of platform specific accessibility APIs. 
```

Comment:
Should this say "DOM Nodes" instead of "Elements"?

Reply:
Researched to see which wording is used elsewhere, and found this definition of "accessible object", which may when matured into a recommendation replace our definition of "included in the accessibility tree": https://www.w3.org/TR/html-aam-1.0/#dfn-accessible-object

Reply:
"accessible object" is also a definition in Core Accessibility API Mappings 1.1, https://www.w3.org/TR/core-aam/#dfn-accessible-object, that is already a recommendation...